src/modules/keyboardmanager/ui/KeyDropDownControl.cpp

```diff
@@ -340,8 +339,8 @@ void KeyDropDownControl::ValidateShortcutFromDropDownList(Grid table, StackPanel
     for (int i = 0; i < keyDropDownControlObjects.size(); i++)
     {
         // Check for errors only if the current selection is a valid shortcut
-        std::vector<DWORD> selectedKeyCodes = KeyboardManagerHelper::GetKeyCodesFromSelectedIndices(keyDropDownControlObjects[i]->GetSelectedIndicesFromStackPanel(parent), GetKeyCodeList(true, colIndex == 1));
-        KeyShortcutUnion currentShortcut;
+        std::vector<int32_t> selectedKeyCodes = GetSelectedCodesFromStackPanel(parent);
+        std::variant<DWORD, Shortcut> currentShortcut;
```

Comment:
[nit] We can use the KeyShortcutUnion alias for std::variant<DWORD, Shortcut>. It was added to make the code more readable

Reply:
done